js/ui/map.js

```diff
@@ -287,6 +287,48 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Set the map's minimum zoom level, and zooms map to that level if it is currently below it. If no parameter provided, unsets the current minimum zoom (sets it to 0)
+     * @param {zoom} any number between 0 and 20
+     * @returns {Map} `this`
+     */
+    setMinZoom: function(minZoom) {
+
+        var zoom = minZoom === null ? 0 minZoom;
+
+        if (zoom <= this.options.maxZoom) {
+            this.options.minZoom = zoom;
+            this._update();
+
+            if (this.getZoom() < zoom) this.setZoom(zoom);
+
+            return this;
+        }
+
+        else throw('minZoom must be less than or equal to the current maxZoom')
+    },
+
+    /**
+     * Set the map's maximum zoom level, and zooms map to that level if it is currently above it. If no parameter provided, unsets the current maximum zoom (sets it to 20)
+     * @param {zoom} any number between 0 and 20
+     * @returns {Map} `this`
+     */
+    setMaxZoom: function(maxZoom) {
+
+        var zoom = maxZoom === null ? 20 : maxZoom;
```

Comment:
make a constant for 20 so that it doesn't lag if someone changes it as the default for the map.